test/new_relic/agent/monitors/trace_context_request_monitor_test.rb

```diff
@@ -101,14 +101,9 @@ def test_does_not_accept_malformed_trace_context
 
       def build_parent_transaction_headers
         carrier = {}
-
-        # stubbing contexted allows trace_context_active? to pass
-        # which, in turn, allows us to insert a trace_context header here.
         parent_txn = in_transaction('referring_txn') do |txn|
-          Agent.instance.stub(:connected?, true) do
```

Comment:
Without some kind of stubbing for `Agent.instance.connected?`, it seems like we're changing the functionality of this test. Is it now `connected?` by default?

Reply:
(Same question for the other removal)

Reply:
I removed the stubs, expecting to have the tests fail and needing me to write a pure Ruby solution (no mocha, no stubs), but they passed. So I just left them passing. Do you think them passing is a case of false positives?

Reply:
Yeah, that's interesting, I would expect these not to work anymore since I think `connected?` is only true after the agent has communicated with the collector. We might want to look into why removing the stubs didn't cause it to fail, seems suspicious to me

Reply:
It could be. I've needed to stub `:connected?` for a truthy value lately. I believe it was in gRPC client, but that's a multiverse test so perhaps it's different. I'm not sure if the test environment changed so that the agent is now connected by default or if it's a false positive.

Reply:
@kaylareopelle @tannalynn 

Ok, so I did the following:

1. Modified `Agent#connected?` to perform `1/0`
2. Ran the `trace_context_request_monitor_test.rb` tests with `rake test file=test/new_relic/agent/monitors/trace_context_request_monitor_test.rb`
3. Ran the `trace_context.rb` perf tests with `bundle exec script/runner -n trace_context`

Everything passed. `Agent#connected?` is never invoked. We don't want no stubs.

Reply:
So weird, I wonder why it was there in the first place 🤔 Maybe something was changed since it was first added that makes it no longer needed? Thanks for checking that out!